src/videostreamindex/VideoSendingAttendee.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export default interface VideoSendingAttendee {
```

Comment:
Why is this interface specific to a `VideoSender`? It's just an attendeeId/externalUserId pair. Suggest adding to `src/attendee` as `Attendee.ts`.

Also why is this an interface and not a regular plain-data class (e.g. https://github.com/aws/amazon-chime-sdk-js/blob/master/src/meetingsession/MeetingSessionConfiguration.ts)?

Reply:
Ok, yes `Attendee` can be more general, I will update `VideoSendingAttendee` -> `Attendee`. 

I wanted to use some interface / type to govern the `{ attendeeId: string; externalId: string }` return type / variable type used at multiple places hence used an interface, is there a pattern we follow in SDK to have all these type related interfaces as plain data classes? I understand plain data classes can be more extensible. I will convert the interface to plain data class. Thanks!